### DISCUSSION TITLE
Does it provide support for NUCLEO-U5A5ZJ-Q?

### DISCUSSION THREAD

**Discussion by HonestQiao** [stm32duino/Arduino_Core_STM32#2172] [Q&A] [answered]
<img width="884" alt="image" src="https://github.com/stm32duino/Arduino_Core_STM32/assets/696708/d30d0975-c68d-4591-809a-f011ebdbdebb">

I got some message from google search.
But after clicking on the search results, I couldn't find the corresponding information.

Link: [https://www.google.com/search?q=add+support+for+U5A5ZJ-Q+gautierg-st](https://www.google.com/search?q=add+support+for+U5A5ZJ-Q+gautierg-st)

@gautierg-st

**Comment by fpistm** [comment, marked as answer]
Hi @HonestQiao 
This item from @gautierg-st is for Zephyr-RTOS not for the STM32 core for Arduino.
For ref: https://github.com/zephyrproject-rtos/zephyr/pull/63144
